### PR TITLE
Mobile menu branch

### PR DESCRIPTION
**Pull request by riskmaris**
Mobile Menu
- Added JavaScript linters
- Added more styling
- Updated HTML file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,11 +20,23 @@
...
           </nav>
+          <div class="x-menu-icon">
+
+            <nav id="x-menu">
+              <i class="bi bi-list" id="menu-bar"></i>
+            </nav>
+            <!---popup navbar-->
+            
+            <i class="bi bi-x-lg" id="x-icon"></i>
+            <ul class="popup-list">
```
Comment (GzimAsani) [inline index.html:33]: - [ ] When the user clicks those options it should move to the correct part of the application, currently your app doesn't do that
https://www.awesomescreenshot.com/video/16626532?key=ed0d29cc9ef9c88242dfc9c6f8d63d30
- [ ] Also remember that after clicking the option and the app sending you to the correct part of the app, the popup should close

File: index.html [hunk 2]
```diff
@@ -20,11 +20,23 @@
...
+              <li><a href="#portfolio">Portfolio</a></li>
+              <li><a href="#about">About</a></li>
+              <li><a href="#contact">Contact</a></li>
```
Comment (GzimAsani) [inline index.html:36]: - [ ] Please adjust the margin/paddings between the options in order to match the design better, also adjust the X with the height and width from the design
Design:
image
Yours:
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @riskmaris                    :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with the hamburger button, it works good :+1: 

 ## Required Changes 

- [ ] Check the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers, and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._



# NOTE: I approved this MR by mistake so the approval will be reverted soon.

**Comment by enniopo** [comment]
@riskmaris the dashboard has been updated and you can now request another code review request once you are ready. 
Thank you and apologies for the inconvenience.

**Comment by GzimAsani** [comment]
Hi @riskmaris, I am glad that your dashboard has been updated. I sincerely apologise for this inconvenience

**Comment by riskmaris** [comment]
Hi
I have made the necessary changes but the problem I have right now is that
I cant make another pull request because when you approved the first one, i
merged it. What should i do?

**Comment by GzimAsani** [comment]
> Hi I have made the necessary changes but the problem I have right now is that I cant make another pull request because when you approved the first one, i merged it. What should i do?
> […](#)

Hi @riskmaris, please ask for a code review on a new PR and mention this PR on a link so the new reviewer can see the required changes that I’ve made

**Pull request merged**